extension/core_functions/scalar/generic/alias.cpp

```diff
@@ -5,7 +5,7 @@ namespace duckdb {
 
 static void AliasFunction(DataChunk &args, ExpressionState &state, Vector &result) {
 	auto &func_expr = state.expr.Cast<BoundFunctionExpression>();
-	Value v(state.expr.alias.empty() ? func_expr.children[0]->GetName() : state.expr.alias);
+	Value v(state.expr.GetAlias().empty() ? func_expr.children[0]->GetName() : state.expr.GetAlias());
```

Comment:
Maybe in the next PR we could consider an `HasAlias()` or so to shorten the often-used `GetAlias().empty()` expression.